playbooks/common/openshift-master/config.yml

```diff
@@ -38,6 +38,11 @@
           public_hostname: "{{ openshift_public_hostname | default(None) }}"
           public_ip: "{{ openshift_public_ip | default(None) }}"
           deployment_type: "{{ openshift_deployment_type }}"
+      - role: nfs
+        local_facts:
+          host: "{{ openshift_nfs_host | default(None) if openshift_nfs_host is defined else groups.oo_nfs_to_config.0 if groups.oo_nfs_to_config | length > 0 else None }}"
```

Comment:
Main goal is not to tie this specifically to NFS...  Eventually we want to be able to easily extend for additional storage providers.